src/CameraZoomPlugin.cc

```diff
@@ -352,31 +393,91 @@ void CameraZoomPlugin::PreUpdate(
   if (!comp)
     return;
 
-  if (!this->impl->zoomChanged)
+  if (this->impl->zoomChanged) {
```

Comment:
Braces on newline, and elsewhere.

- https://gazebosim.org/docs/all/contributing#style-guides

Reply:
Seems fixed now from your PR. Thanks for the room service!